extensions/wikia/PremiumPageHeader/scripts/article-header.js

```diff
@@ -0,0 +1,20 @@
+require(['wikia.window', 'jquery', 'wikia.tracker'], function (window, $, tracker) {
+	'use strict';
+
+	var _track = tracker.buildTrackingFunction({
```

Comment:
why the name of this var starts with underscore? is it some new convention? AFAIR (at least in tracking) we used just `track`